.storybook/main.js

```diff
@@ -1,5 +1,8 @@
+// TODO: fix No props found when using docs addon
+const path = require("path");
+
 module.exports = {
-  stories: ["../src/**/*.stories.tsx"],
+  stories: ["../src/**/*.stories.(ts|tsx|mdx)"],
```

Comment:
what's the `mdx`? and is it used in our stories?

Reply:
We currently don't have a combined markdown in any Stories, but we will need it later.
reference: https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/mdx.md